main/ahb_for_sles.go

```diff
@@ -97,61 +98,88 @@ func parseCfg(filename string) (map[string]map[string]string, error) {
 	return ini, nil
 }
 
-func _isRegistered() (bool, error) {
-	services, err := filepath.Glob("/etc/zypp/services.d/*.service")
-	if err != nil {
-		fmt.Fprintln(os.Stderr, err)
-		return false, err
+func _getSUSEConnectStatus() (bool, bool, error) {
```

Comment:
The function name is suspect. `_getSUSEConnectStatus` should return a status, based on name and should not have any side effects. But the side effect of this call is that, depending on the state of the system, repositories may get enabled. We should strive for side effect free implementations.